sdk/keyvault/azure-keyvault-keys/tests/test_samples_keys.py

```diff
@@ -42,7 +42,7 @@ class TestExamplesKeyVault(KeyVaultTestCase, KeysTestCase):
     @recorded_by_proxy
     def test_example_key_crud_operations(self, key_client, **kwargs):
         if (self.is_live and os.environ["KEYVAULT_SKU"] != "premium"):
-            pytest.skip("This test not supprot in usgov/china region. Follow up with service team")
+            pytest.skip("This test is not supported on standard SKU vaults. Follow up with service team")
```

Comment:
```suggestion
            pytest.skip("This test is not supported in usgov/china region. Follow up with service team")
```

Reply:
My thinking was that the check here is specifically for standard SKU vaults, rather than the cloud. China cloud tests do only use standard SKUs, so the message is technically accurate, but the incompatibility seems to come more from the SKU than from the region.

What do you think?